packages/css/src/alert/alert.scss

```
@@ -0,0 +1,78 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-alert {
+  align-items: flex-start;
+  background-color: var(--amsterdam-alert-background-color);
+  border-color: var(--amsterdam-alert-border-color);
+  border-style: var(--amsterdam-alert-border-style);
+  border-width: var(--amsterdam-alert-border-width);
+  display: flex;
+  flex-direction: row;
+  gap: var(--amsterdam-alert-gap);
+  justify-content: space-between;
+  padding-block-end: var(--amsterdam-alert-padding-block-end);
+  padding-block-start: var(--amsterdam-alert-padding-block-start);
+  padding-inline-end: var(--amsterdam-alert-padding-inline-end);
+  padding-inline-start: var(--amsterdam-alert-padding-inline-start);
+}
+
+.amsterdam-alert__content {
+  flex-basis: 100%;
```

Prefer using the shorthand property, as [advised in the spec](https://drafts.csswg.org/css-flexbox/#flex-components).

```suggestion
  flex: auto;
```